cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -173,7 +174,8 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         val repository = artifactory.repository(repoName)
         try {
             repository.delete(releaseName_build)
-        } catch (_: Exception) {
+        } catch (e: Exception) {
+            logger.error("Failed to delete folder {}: {}", releaseName_build, e.message)
```

Based on the guidelines, I need to add a meaningful exception handling method within the specified code range. The strategy suggests using logging to capture and log details about the exception encountered during the delete operation. Since the available dependencies suggest that `kotlin-logging` is an allowed dependency, I assume it's currently set up or easy to set up in the project. I replaced the ignored exception with a logging statement that logs the error at the ERROR level, capturing the exception's message for future debugging and visibility into any issues that arise during the deletion operation.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)